profiles/common/modules/features/multisite_translations/multisite_translations.module

```diff
@@ -198,17 +205,17 @@ function create_translation_roles() {
         'submit translation jobs',
         'accept translation jobs',
         'access workbench',
-        //'administer tmgmt',
+        // 'administer tmgmt',.
```

Comment:
This should be removed, commented out code is not permitted in the coding standards.